### PR TITLE
Chart: update 'containerBackgroundColor' description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md [hunk 1]
```diff
@@ -8,7 +8,10 @@ default: '#FFFFFF'
 Colors the background of the chart container.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md]: > Colors the background of **the** chart container.

The first and most important question -- of what chart container do we speak? From what I see, we do not describe a property of a hypothetical chart container class, therefore, we cannot refer to "the chart container" as the [current] container instance.

Can we say that the parent property set (`Options`) belongs to the chart container? Or can we say that the Options property set belongs to the [current] chart control that uses the said container?

I believe that my assumption is correct but feel free to object if I misinterpreted your scenario.

My current take on this short description is the following:

```suggestion
Specifies background color of the chart container.
```

Yes, we could interpret "the background color" as the [current] background color. However, according to the newest guideline on article usage, we should keep definite article case occurrences to a minimum.

In this scenario, we can omit article when it's related to an attribute -- such as color, width, length, etc.

File: api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md [hunk 2]
```diff
@@ -8,7 +8,10 @@ default: '#FFFFFF'
...
 
 ---
-If the background of your page has a color different from white, assign this color to the **containerBackgroundColor** property to ensure that the UI component will blend smoothly into your page. Certain UI component elements, such as series points in the [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), will use this color when coordinating with the page.
+The container is a `<div>` element that contains Chart. Certain Chart elements, such as series points in the [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), use the same color as a background color of the container. You may need this property if the background of your page has a color different from white. Assign this color to the **containerBackgroundColor** property to ensure that the Chart elements are color-coordinated with your page.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md]: > **The** container is a `<div>` element that contains Chart.

Yes, we probably could refer to "the container" as the container mentioned in your short description earlier on. However, this is just an introductory sentence that just explains the concept of chart containers. Therefore, this sentence should speak in generalities, that is, in indefinite article cases.

>  Certain Chart elements, such as series points in **the** [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode')

Remember the pattern -- in X mode, in Y format, in Z state, in W order? Every time we use a named mode/format/state/order as an aspect of circumstances/addendum to the base SVO sentence structure -- no article is necessary.


> Assign this color to the **containerBackgroundColor** property to ensure that **the** Chart elements are color-coordinated with your page.

Why did you use a definite article here? Is it really important in this context? I don't believe so.

```suggestion
A chart container is a `<div>` element that contains a Chart control. Certain Chart elements, such as series points in [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), have the same color as the container's background color. You may need to change this property value if the your page's background color differs from white. Assign this color to the **containerBackgroundColor** property to ensure that Chart elements are colored consistently with your page.
```

File: api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md [hunk 3]
```diff
@@ -8,7 +8,10 @@ default: '#FFFFFF'
...
+
+If you need to define a background color for the Chart itself, you can use the [elementAttr](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#elementAttr) property to assign a custom CSS class with the background color specified.
```
Comment (dmitry-eliseev-devexpress) [inline api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md]:
```suggestion
If you need to define a background color for the Chart itself, you can use the [elementAttr](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#elementAttr) property to assign a custom CSS class with a defined background color.
```

**Review by dmitry-eliseev-devexpress — APPROVED** [review]
As always, I commend your efforts...

I understand that the new article guideline complicated things a lot for you. However, please do understand the main point -- we need to minimize usage of definite article cases. Every definite article case defined in any manner -- it's an emphasis, a stress mark. It's not unlike formatting -- when everything is emphasized -- nothing is emphasized.

I tried to explain every incorrect or questionable article case, however -- should I misinterpret something or you need additional answers or explanations -- you need only ask.

Other than that -- I'm approving this merge request.

File: api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md [hunk 4]
```diff
@@ -5,10 +5,13 @@ default: '#FFFFFF'
...
+Specifies background color of the chart container.
 
 ---
-If the background of your page has a color different from white, assign this color to the **containerBackgroundColor** property to ensure that the UI component will blend smoothly into your page. Certain UI component elements, such as series points in the [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), will use this color when coordinating with the page.
+A chart container is a `<div>` element that contains a Chart control. Certain Chart elements, such as series points in [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), have the same color as the container's background color. You may need to change this property value if the your page's background color differs from white. Assign this color to the **containerBackgroundColor** property to ensure that Chart elements are colored consistently with your page.
```
Comment (vladaskorohodova) [inline api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md]:
```suggestion
A chart container is a `<div>` element that contains a Chart control. Certain Chart elements, such as series points in [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), have the same color as the container's background color. You may need to change this property value if your page's background color differs from white. Assign this color to the **containerBackgroundColor** property to ensure that Chart elements are colored consistently with your page.
```

File: api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md [hunk 5]
```diff
@@ -5,10 +5,13 @@ default: '#FFFFFF'
...
+Specifies background color of the chart container.
 
 ---
-If the background of your page has a color different from white, assign this color to the **containerBackgroundColor** property to ensure that the UI component will blend smoothly into your page. Certain UI component elements, such as series points in the [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), will use this color when coordinating with the page.
+A chart container is a `<div>` element that contains a Chart control. Certain Chart elements, such as series points in [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), have the same color as the container's background color. You may need to change this property value if your page's background color differs from white. Assign this color to the **containerBackgroundColor** property to ensure that Chart elements are colored consistently with your page.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md]:
```suggestion
A chart container is a `<div>` element that contains a Chart control. Certain Chart elements, such as series points in [hovered state](/api-reference/10%20UI%20Components/dxChart/5%20Series%20Types/CommonSeries/point/hoverMode.md '/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/point/#hoverMode'), have the same color as the container's background color. You may need to change this property value if your page's background color differs from white. Assign this color to the **containerBackgroundColor** property to ensure that the color of Chart elements is consistent with your page.
```

File: api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md [hunk 6]
```diff
@@ -5,10 +5,13 @@ default: '#FFFFFF'
...
+
+If you need to define a background color for the Chart itself, you can use the [elementAttr](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#elementAttr) property to assign a custom CSS class with a defined background color.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Components/dxChart/1 Configuration/containerBackgroundColor.md]:
```suggestion
If you need to define a background color for the Chart, you can use the [elementAttr](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#elementAttr) property to assign a custom CSS class with a defined background color.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**